### PR TITLE
Core 581

### PR DESCRIPTION
**Pull request by tundeaoni**
https://remerge.atlassian.net/browse/CORE-581

### CODE REVIEW

**Comment by remerge-hal** [comment]
This comment ensures that the correct Slack channel is notified after the team/project label `CORE` has been added to this pull request.

See [this comment](https://remerge.atlassian.net/browse/ITSUP-1973?focusedCommentId=90701) for details.

File: .github/workflows/nomad-pack.yml [hunk 1]
```diff
@@ -53,6 +57,9 @@ on:
       slack_bot_token:
         required: true
         description: "Access token for Slack"
+      op_service_account:
```
Comment (hollow) [inline .github/workflows/nomad-pack.yml:60]: let's rename this to `op_service_account_token`

File: .github/workflows/nomad-pack.yml [hunk 2]
```diff
@@ -71,6 +78,30 @@ jobs:
...
+
+      - name: Load nomad_token from 1password
+        if: ${{ inputs.one_password_secret_path && steps.secret-check.outputs.nomad_token_set == 'false' }}
+        uses: 1password/load-secrets-action@v1
+        with:
+          # Export loaded secrets as environment variables
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.op_service_account }}
+          NOMAD_TOKEN: ${{ format('op://{0}', inputs.one_password_secret_path) }}
```
Comment (hollow) [inline .github/workflows/nomad-pack.yml:104]: the `op://` prefix should either be part of the input (and then we can skip the format here) or we change the input to just the item name and hardcode the vault `secrets` as well (the workflows don't have access to any other vault anyway)

**Review by hollow — CHANGES_REQUESTED** [review]

**Comment by tundeaoni** [comment]
@hollow PR has been updated with your suggestions

**Pull request merged**